drivers/mssqql.go

```diff
@@ -275,19 +276,36 @@ func (db *MSSQL) GetRecords(database, table, where, sort string, offset, limit i
 		if err := rows.Scan(rowValues...); err != nil {
 			return nil, 0, err
 		}
-
+		// Get column types to identify UNIQUEIDENTIFIER
+		columnTypes, err := rows.ColumnTypes()
+		if err != nil {
+			return nil, 0, err
+		}
 		var row []string
-		for _, col := range rowValues {
+		for i, col := range rowValues {
 			if col == nil {
 				row = append(row, "NULL&")
 				continue
 			}
 
-			colval := string(*col.(*sql.RawBytes))
-			if colval == "" {
-				row = append(row, "EMPTY&")
+			rawBytes := *col.(*sql.RawBytes)
+			colType := columnTypes[i].DatabaseTypeName()
```

Comment:
I would add security checks here


```suggestion
			rawBytes, ok := *col.(*sql.RawBytes)
			if !ok {
				return nil, 0, errors.New("unexpected whatever")
			}
			columnType, ok := columnTypes[i]
			if !ok {
				return nil, 0, errors.New("unexpected whatever")
			}
			colType := columnType.DatabaseTypeName()
```

Reply:
I tried this suggestion but I got those errors
Can you help out?
I started like yesterday with go and I don't know how to fix this.

<img width="931" alt="image" src="https://github.com/user-attachments/assets/6dee41ef-ab0f-4a3e-9e49-7288b7062baa" />

Reply:
Ok, nevermind I fixed that one